src/pages/create_blog.jsx

```diff
@@ -0,0 +1,158 @@
+                        placeholder='write the Tiltle Here'
+                        onChange={(e) => SetEnTitle(e.target.value)}
+                    />
+
+                    <label className='block mb-2 text-sm font-medium text-gray-900 dark:text-white'>
+                        Article in English{" "}
+                    </label>
+                    <textarea
+                        rows='8'
+                        className='block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
+                        placeholder='write the Article Here'
+                        onChange={(e) => setEnPostText(e.target.value)}
+                    ></textarea>
+                </div>
+
+                <div className='w-full md:w-1/2 p-4 bg-light-cyan'>
+                    <label
+                        className='block mb-2 text-sm font-medium text-gray-900 dark:text-white'
+                        dir='rtl'
+                    >
+                        العنوان بالعربية
+                    </label>
+                    <input
+                        type='text'
+                        className='block w-full p-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 sm:text-xs focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
+                        placeholder='أكتب العنوان هنا'
+                        dir='rtl'
+                        onChange={(e) => setArTitle(e.target.value)}
+                    />
+                    <label
+                        className='block mb-2 text-sm font-medium text-gray-900 dark:text-white'
+                        dir='rtl'
+                    >
+                        المقال بالعربية
+                    </label>
+                    <textarea
+                        id='message'
+                        rows='8'
+                        className='block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500'
+                        placeholder='أكتب مقالك هنا'
+                        dir='rtl'
+                        onChange={(e) => setArPostText(e.target.value)}
+                    ></textarea>
+
+                    <label
+                        className='block mb-2 text-sm font-medium text-gray-900 items-center'
+                        for='file_input'
+                    >
+                        {t("UploadArticleImage")}
+                    </label>
+                    <input
```

Comment:
refactor this code by using input component instead of building it

Reply:
already edit it ,done